android/src/main/java/com/magicleap/magicscript/scene/nodes/dropdown/UiDropdownListItemNode.kt

```diff
@@ -0,0 +1,98 @@
+/*
+ *  Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package com.magicleap.magicscript.scene.nodes.dropdown
+
+import android.graphics.Typeface
+import android.os.Bundle
+import com.facebook.react.bridge.ReadableMap
+import com.magicleap.magicscript.font.FontParams
+import com.magicleap.magicscript.font.FontProvider
+import com.magicleap.magicscript.font.FontStyle
+import com.magicleap.magicscript.font.FontWeight
+import com.magicleap.magicscript.scene.nodes.base.TransformNode
+import com.magicleap.magicscript.utils.read
+
+open class UiDropdownListItemNode(
+    initProps: ReadableMap,
+    private val fontProvider: FontProvider
+) : TransformNode(initProps, false, false) {
+
+    companion object {
+        const val PROP_ID = "id"
+        const val PROP_LABEL = "label"
+        const val PROP_SELECTED = "selected"
+
+        const val MAX_CHARACTERS_UNLIMITED = 0
+    }
+
+    var onSelectionChangeRequest: ((select: Boolean) -> Unit)? = null
+
+    var id: Int = 0
+        private set
+
+    var label: String = ""
+        private set
+
+    var isSelected = false
```

Comment:
I'd like to rename it to just `selected` as kt generate `is..` functions as a boolean getter